framegear/.gitignore

```diff
@@ -0,0 +1,36 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
```

Comment:
nit. for next PR, do we need this? My guess is not

Reply:
ah probably not, was autogenerated. will clean up on the next one

Reply:
following up on this, worth including because after removing it there are ~1000 files git tries to track. Updating the outer `.gitignore` with this stuff would probably be confusing so going to keep it